client-src/utils/log.js

```diff
@@ -27,8 +27,8 @@ const logEnabledFeatures = (features) => {
   let logString = "Server started:";
 
   // Server started: Hot Module Replacement enabled, Live Reloading enabled, Overlay disabled.
-  for (const [key, value] of Object.entries(features)) {
-    logString += ` ${key} ${value ? "enabled" : "disabled"},`;
+  for (const key of Object.keys(features)) {
```

Comment:
Still new syntax https://caniuse.com/mdn-javascript_statements_for_of, let rewrite on simple loop

Reply:
Actually, this will be converted to for loop by babel:

<img width="822" alt="Screenshot 2022-08-27 at 6 51 03 AM" src="https://user-images.githubusercontent.com/46647141/187008793-81c0de87-8256-4c60-9cfb-b111e04e1163.png">

Reply:
Fixed.